src/cascadia/TerminalApp/SettingsTab.h

```diff
@@ -26,8 +26,8 @@ namespace winrt::TerminalApp::implementation
     struct SettingsTab : SettingsTabT<SettingsTab>
     {
     public:
-        SettingsTab();
-
+        SettingsTab() = delete;

```

Comment:
I'm 90% sure that declaring another constructor and not declaring the empty one is the same as declaring the empty one as `=delete;`

Reply:
Unless WinRT does something I'm not aware of...

Reply:
yup you're right about that